services/ui-src/package.json

```diff
@@ -97,10 +89,6 @@
       "src"
     ],
     "verbose": true,
-    "moduleNameMapper": {
```

Comment:
I'm not sure if I should have removed this, but everything seems to work without.